packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -841,73 +841,71 @@ curl -v -X GET \
-            "allow": [
-              "GET"
-            ]
-          }
+        "status": "NEXT",
+        "alg": "RS256",
+        "e": "AQAB",
+        "n": "l1hZ_g2sgBE3oHvu34T-5XP18FYJWgtul_nRNg-5xra5ySkaXEOJUDRERUG0HrR42uqf9jYrUTwg9fp-SqqNIdHRaN8EwRSDRsKAwK
+        3 HIJ2NJfgmrrO2ABkeyUq6rzHxAumiKv1iLFpSawSIiTEBJERtUCDcjbbqyHVFuivIFgH8L37 - XDIDb0XG - R8DOoOHLJPTpsgH - rJe
+        M5w96VIRZInsGC5OGWkFdtgk6OkbvVd7_TXcxLCpWeg1vlbmX - 0 TmG5yjSj7ek05txcpxIqYu - 7 FIGT0KKvXge_BOSEUlJpBhLKU28
+        OtsOnmc3NLIGXB - GeDiUZiBYQdPR - myB4ZoQ ",
+        "kid": "Y3vBOdYT-l-I0j-gRQ26XjutSX00TeWiSguuDhW3ngo",
+        "kty": "RSA",
+        "use": "sig",
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/authorizationServers/ausnsopoM6vBRB3PD0g3/credentials/keys/Y3vBOdYT-l-I0j-gRQ26XjutSX00TeWiSguuDhW3ngo",
+                "hints": {
+                    "allow": [
+                        "GET"
+                    ]
+                }
+            }
         }
-      }
     },
     {
-      "status": "EXPIRED",
-      "alg": "RS256",
-      "e": "AQAB",
-      "n": "lC4ehVB6W0OCtNPnz8udYH9Ao83B6EKnHA5eTcMOap_lQZ-nKtS1lZwBj4wXRVc1XmS0d2OQFA1VMQ-dHLDE3CiGfsGqWbaiZFdW7U
-            GLO1nAwfDdH6xp3xwpKOMewDXbAHJlXdYYAe2ap-CE9c5WLTUBU6JROuWcorHCNJisj1aExyiY5t3JQQVGpBz2oUIHo7NRzQoKimvp
-            dMvMzcYnTlk1dhlG11b1GTkBclprm1BmOP7Ltjd7aEumOJWS67nKcAZzl48Zyg5KtV11V9F9dkGt25qHauqFKL7w3wu-DYhT0hmyFc
-            wn-tXS6e6HQbfHhR_MQxysLtDGOk2ViWv8AQ",
-      "kid": "h5Sr3LXcpQiQlAUVPdhrdLFoIvkhRTAVs_h39bQnxlU",
-      "kty": "RSA",
-      "use": "sig",
-      "_links": {
-        "self": {
-          "href": "https://{yourOktaDomain}/api/v1/authorizationServers/ausnsopoM6vBRB3PD0g3/credentials/keys/h5Sr3LXcpQiQlAUVPdhrdLFoIvkhRTAVs_h39bQnxlU",
-          "hints": {
-            "allow": [
-              "GET"
-            ]
-          }
+        "status": "EXPIRED",
+        "alg": "RS256",
+        "e": "AQAB",
+        "n": "lC4ehVB6W0OCtNPnz8udYH9Ao83B6EKnHA5eTcMOap_lQZ-nKtS1lZwBj4wXRVc1XmS0d2OQFA1VMQ-dHLDE3CiGfsGqWbaiZFdW7U
+        GLO1nAwfDdH6xp3xwpKOMewDXbAHJlXdYYAe2ap - CE9c5WLTUBU6JROuWcorHCNJisj1aExyiY5t3JQQVGpBz2oUIHo7NRzQoKimvp
+        dMvMzcYnTlk1dhlG11b1GTkBclprm1BmOP7Ltjd7aEumOJWS67nKcAZzl48Zyg5KtV11V9F9dkGt25qHauqFKL7w3wu - DYhT0hmyFc
+        wn - tXS6e6HQbfHhR_MQxysLtDGOk2ViWv8AQ ",
+        "kid": "h5Sr3LXcpQiQlAUVPdhrdLFoIvkhRTAVs_h39bQnxlU",
+        "kty": "RSA",
+        "use": "sig",
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/authorizationServers/ausnsopoM6vBRB3PD0g3/credentials/keys/h5Sr3LXcpQiQlAUVPdhrdLFoIvkhRTAVs_h39bQnxlU",
+                "hints": {
+                    "allow": [
+                        "GET"
+                    ]
+                }
+            }
         }
-      }
-    },
-  ]
-}
+    }
+]
 ```
 
 * The listed `ACTIVE` key is used to sign tokens issued by the authorization server.
```

Comment:
* The listed `ACTIVE` key is used to sign tokens issued by the Authorization Server.

Reply:
👍